test/unit/alignment/pairwise/pairwise_alignment_collection_test_template.hpp

```diff
@@ -48,24 +48,33 @@ TYPED_TEST_P(pairwise_alignment_collection_test, score)
                     fixture.get_scores());
 }
 
-TYPED_TEST_P(pairwise_alignment_collection_test, back_coordinate)
+TYPED_TEST_P(pairwise_alignment_collection_test, end_position_pair)
 {
     auto const & fixture = this->fixture();
-    seqan3::configuration align_cfg = fixture.config | seqan3::align_cfg::result{seqan3::with_back_coordinate};
+    seqan3::configuration align_cfg = fixture.config | seqan3::align_cfg::result{seqan3::with_end_position_pair};
     auto [database, query] = fixture.get_sequences();
     auto res_vec = seqan3::align_pairwise(seqan3::views::zip(database, query), align_cfg)
                  | seqan3::views::to<std::vector>;
 
     EXPECT_TRUE((std::ranges::equal(res_vec | std::views::transform([] (auto res) { return res.score(); }),
                                     fixture.get_scores())));
-    EXPECT_TRUE((std::ranges::equal(res_vec | std::views::transform([] (auto res) { return res.back_coordinate(); }),
-                                    fixture.get_back_coordinates())));
+    EXPECT_TRUE((std::ranges::equal(res_vec |
+                                    std::views::transform([] (auto res)
+                                                          {
+
+                                                           return seqan3::alignment_coordinate{
+                                                                     seqan3::detail::column_index_type{
+                                                                         res.sequence1_end_position()},
+                                                                     seqan3::detail::row_index_type{
+                                                                         res.sequence2_end_position()}};
+                                                          }),
```

Comment:
```suggestion
    EXPECT_TRUE((std::ranges::equal(res_vec |
                                    std::views::transform([] (auto res)
                                    {
                                         return seqan3::alignment_coordinate{
                                                   seqan3::detail::column_index_type{res.sequence1_end_position()},
                                                   seqan3::detail::row_index_type{res.sequence2_end_position()}};
                                    }),
```

I think this is also fine.

Reply:
There are some more occurrence of this below.